examples/sol-contract/src/processor.rs

```diff
@@ -0,0 +1,67 @@
+//! Program instruction processor
+
+use solana_program::msg;
+use solana_program::pubkey::Pubkey;
+use solana_program::account_info::AccountInfo;
+use solana_program::entrypoint::ProgramResult;
+use solana_program::program_error::ProgramError;
+
+use borsh::BorshDeserialize;
+use crate::instruction::PythClientInstruction;
+use pyth_sdk_solana::load_price_feed_from_account_info;
+
+pub fn process_instruction(
+    _program_id: &Pubkey,
+    _accounts: &[AccountInfo],
+    input: &[u8],
+) -> ProgramResult {
+    // Checking the validity of parameters is important.
+    // This important step is skipped in this example contract.
```

Comment:
I think you should do these checks in this example for completeness. As this stands, it will show people how to use the prices in an unsafe way.

Reply:
One way to do this would be to add a `configure` instruction that stores the loan / collateral price feed account keys in the program somewhere. You can probably skip permissions on the `configure` instruction for the example (but comment that real code should only let admins configure this)